### PR TITLE
CSS-7423 Scaling integration testing for Trino charm

### PR DESCRIPTION
**Pull request by AmberCharitos**
Add scaling integration test for worker

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -51,6 +51,20 @@ deps =
 commands =
     pytest {[vars]tst_path}integration/test_policy.py -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
 
+[testenv:integration-scaling]
```
Comment (gtato) [inline tox.ini:54]: i guess we can group these integration tests together in one
```
[testenv]
...
charm: TEST_FILE=test_charm.py
policy: TEST_FILE=test_policy.py
scaling: TEST_FILE=test_policy.py


[testenv:integration-{charm, policy, scaling}]
description = Run integration tests
deps =
    ipdb==0.13.9
    juju==3.1.0.1
    pytest==7.4.0
    pytest-operator==0.29.0
    pytest-asyncio==0.21.0
    trino==0.326.0
    apache-ranger==0.0.11
    -r{toxinidir}/requirements.txt
commands =
    pytest {[vars]tst_path}integration/{env:TEST_FILE} -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}

```

Comment (AmberCharitos) [reply tox.ini]: Updated on both PRs thank you.

**Review by kelkawi-a — APPROVED** [review]
LGTM

**Review by gtato — APPROVED** [review]
lg

**Pull request merged**